docs/Manifest.toml

```diff
@@ -1,6 +1,6 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.10.7"
+julia_version = "1.10.5"
```

Comment:
I can update Julia and change it back. But on clima cluster it's still 1.10.5 so if I update dependencies there it will be changed to 1.10.5. Not sure what is the best practice?

Reply:
Anything is fine as long as you don't change the minor version. I think it's nice to keep what was already here (not necessarily what's on the clusters) to reduce git diff noises